src/components/Dashboard/TodoList.vue

```diff
@@ -260,22 +260,27 @@
                   v-else-if="EnumUtilities.isStatusChangeRequested(item)"
                   class="todo-subtitle pt-1"
                 >
-                  <span class="orange--text text--darken-2">CHANGE REQUESTED</span>
+                  <span class="orange--text text--darken-2 font-weight-bold">CHANGE REQUESTED</span>
                   <span class="vert-pipe" />
-                  <span>PAID (filed by {{ item.submitter }} on <DateTooltip :date="item.submittedDate" />)</span>
+                  <span>Submitted by {{ item.submitter }} on <DateTooltip :date="item.submittedDate" />)</span>
                 </div>
 
                 <!-- approved continuation filing -->
                 <div
                   v-else-if="EnumUtilities.isStatusApproved(item)"
-                  class="todo-subtitle pt-1"
+                  class="todo-list-detail pt-1"
                 >
-                  <span
+                  <span class="list-item__subtitle">
+                    Continuation Authorization APPROVED |
+                    Filed by {{ item.submitter }} on <DateTooltip :date="item.submittedDate" />
+                  </span>
+                  <p
                     v-if="getNameRequest"
                     class="list-item__subtitle"
+                    style="margin-bottom: 0"
                   >
-                    NR APPROVED - {{ expiresText(getNameRequest) }}
-                  </span>
+                    Name Request APPROVED - {{ expiresText(getNameRequest) }}
```

Comment:
It's possible the NR has expired by now.

(It's true that Filings UI won't show you the dashboard if the NR has expired, but we might change that, and then this would display incorrectly.)

Reply:
It seems when the NR has expired, getNameRequest returns null. 
I tried with a Pending Review item: https://business-filings-dev--pr-697-l4j2px9g.web.app/TqPo3HWiXN

Reply:
It looks like the NR data was not fetched for this Awaiting Review continuation authorization (according to the network traffic).

I think there's code in App.vue that fetches NRs in specific scenarios. This scenario may be missing.

Reply:
- I added a method to check whether the NR expires - line 284.
- I updated APP.vue, now NR is loaded for Pending List

Reply:
Now, with https://business-filings-dev--pr-697-l4j2px9g.web.app/TqPo3HWiXN, I get the expected name request expiration dialog.

Your solution of not showing the NR info if it's expired is OK for now. Ultimately, if we do show the (new) business dashboard with an expired NR, we can revisit this.

Resolving.